apps/passport-client/components/shared/PCDCard.tsx

```diff
@@ -170,9 +175,9 @@ function CardBody({
   }
 
   if (pcdCollection.hasPackage(pcd.type)) {
-    const pcdPackage = pcdCollection.getPackage(pcd.type);
-    if (pcdPackage.renderCardBody) {
-      const Component = pcdPackage.renderCardBody;
+    const ui = getUI(pcd.type);
+    if (ui) {
+      const Component = ui.renderCardBody;
       return <Component pcd={pcd} />;
     }
   }
```

Comment:
wonder if it's worth adding several `console.warn` here in the case that a pcd package isn't found